src/tests/class_tests/openms/source/AScore_test.cpp

```diff
@@ -26,7 +26,7 @@ class AScoreTest : public AScore
       return computeSiteDeterminingIons_(th_spectra, candidates, site_determining_ions);
     }
 
-    std::vector<Size> getSitesTest_(const AASequence& without_phospho) const {
+    std::vector<Size> getSitesTest_(String without_phospho) const {
```

Comment:
Would const String& work? In c++ this would lead to a copy of string (potentially slow)